### DISCUSSION TITLE
Option to roll back to first letter of mistyped word

### DISCUSSION THREAD

**Discussion by lightbluepoppy** [monkeytypegame/monkeytype#4035] [Ideas] [answered]
There are two options when mistyped; 
(1) stop at letter and 
(2) stop at word (basically you must just correct the mistake) 

I'd love to get another option which is
(3) resets the word and roll back to the first letter

This allows me to learn how to type the specific word efficiently. Currently, I must type backspace when I typed incorrectly which is not quite efficient.

**Comment by Miodec** [comment, marked as answer]
I would recommend just learning ctrl + backspace. In real typing, the word won't delete itself if you mistype it.

Also, deleting the word completely would be quite confusing imo.
